sdk/eventgrid/eventgrid-namespaces/src/index.ts

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+export { AzureKeyCredential } from "@azure/core-auth";
+
+export {
+  CloudEvent,
+  ReceiveResult,
+  ReceiveDetails,
+  BrokerProperties,
+  AcknowledgeResult,
+  FailedLockToken,
+  ReleaseResult,
+  RejectResult,
+  PublishCloudEventsOptions,
+  ReceiveCloudEventsOptions,
+  AcknowledgeCloudEventsOptions,
+  ReleaseCloudEventsOptions,
+  RejectCloudEventsOptions,
+  ReleaseDelay,
+  RenewCloudEventLocksOptions,
+  RenewCloudEventLocksResult,
+} from "./cadl-generated/models";
+
+export { EventGridNamespacesClient as EventGridClient } from "./eventGridNamespacesClient";
+
+export { EventGridClientOptions } from "./cadl-generated";
+
+export { PublishResultOutput } from "./cadl-generated/rest";
+
+export { OperationOptions as OperationRestOptions } from "@azure-rest/core-client";
```

Comment:
`OperationRestOptions` doesn't sound good.  I wonder whether we have existing pattern or not.  Like `RestLevelOperationOptions`?

Reply:
Changed it to OperationOptions itself. (in the next commit)